README.md

```diff
@@ -75,6 +75,70 @@ $ jsonnet -J vendor example.jsonnet
 
 ### Customizing
 
+#### Adding dashboards
+
+This setup is optimized to work best when Grafana is used declaratively, so when adding dashboards they are added declaratively as well. In jsonnet there are libraries available to avoid having to repeat boilerplate of Grafana dashboard json. An example with the [grafana/grafonnet-lib](https://github.com/grafana/grafonnet-lib):
+
+[embedmd]:# (examples/dashboard-definition.jsonnet)
+```jsonnet
+local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
+local service = k.core.v1.service;
+local servicePort = k.core.v1.service.mixin.spec.portsType;
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+
+local grafana = ((import 'grafana/grafana.libsonnet') + {
```

Comment:
```suggestion
local grafana = 
  ((import 'grafana/grafana.libsonnet') + {
```
I've previously talked to @squat about this: Can we make a new line so we don't have this weird indentations?
/cc @s-urbaniak

Reply:
:+1: I believe I follow the proposed idiom already here: https://github.com/openshift/cluster-monitoring-operator/commit/534c4fe68a64708253e6cdc518a049ab3886067d#diff-254ce52fe46e44a39a4644a7bf612a2

Reply:
yes makes sense, please take a look